docs/content/docs/dev/datastream/fault-tolerance/checkpointing.md

```diff
@@ -230,19 +230,19 @@ Flink currently only provides processing guarantees for jobs without iterations.
 
 Please note that records in flight in the loop edges (and the state changes associated with them) will be lost during failure.
 
-## Checkpointing with parts of the graph finished *(BETA)*
+## Checkpointing with parts of the graph finished
```

Comment:
Do we have documented somewhere that finished operators are waiting for the final checkpoint to complete before closing? If not, can you add such section that? It should also explicitly state that it can prolong completion of the job, until that last final checkpoint is triggered, so longer checkpointing intervals can block job from completing.